### PR TITLE
Update installing-ocp-agent-inputs.adoc

### PR DESCRIPTION
**Pull request by Ultimate-etamitlU**
Fixing the issue #66031

Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skopacz1** [comment]
@bmanzari could you please take a look at this PR and let us know what you think? Thanks!

**Review by skopacz1 — APPROVED** [review]
LGTM from a peer review perspective

**Comment by skopacz1** [comment]
@Ultimate-etamitlU thanks for your contribution to our docs!

**Comment by jab-rh** [comment]
@skopacz1, do we know what versions this applies to?

**Comment by skopacz1** [comment]
@jab-rh I'm imagining to all versions where this module exists, so 4.13+, but let me confirm that with an SME/QE. In any case, this PR now needs a rebase, so feel free to remove the merge review label if you'd like me to get everything sorted before requesting another merge review.

**Comment by skopacz1** [comment]
Sorry @Ultimate-etamitlU but would you mind rebasing this PR to get rid of the conflicts? I think one of my PRs just merged and caused this issue.

**Comment by skopacz1** [comment]
Just confirmed with QE that this can be applied to 4.13+

File: modules/installing-ocp-agent-inputs.adoc [hunk 1]
```diff
@@ -170,24 +169,3 @@ You must provide the rendezvous IP address when you do not specify at least one
...
-rendezvousIP: 192.168.111.80
-bootArtifactsBaseURL: <asset_server_URL>
-----
-+
-Where `<asset_server_URL>` is the URL of the server you will upload the PXE assets to.
-endif::pxe-boot[]
-
-ifeval::["{context}" == "prepare-pxe-assets-agent"]
-:!pxe-boot:
-endif::[]
```
Comment (skopacz1) [inline modules/installing-ocp-agent-inputs.adoc:193]: This is content that I merged earlier, which triggered the conflict, but this should be included in the main branch of the docs. Can you add this content back in, and then squash all of your commits into one?

**Review by Ultimate-etamitlU — COMMENTED** [review]
Merging all in one

**Comment by Ultimate-etamitlU** [comment]
Tried by GUI, don't have CLI so will be adding up it tomorrow. Not fond of CLI that much so may be missing something over GUI to collect everything at a place.

**Comment by Ultimate-etamitlU** [comment]
Closing this one as there are multiple changes I have on a single file and would be adding a fresh PR to avoid confusion here. 
Sorry for little noise but rebasing from GUI or dealing with PR's raise from GUI and dealing with those with CLI is expert task :/

**Pull request closed without merging**